ocean-client/src/programs/common-program.ts

```diff
@@ -297,6 +454,12 @@ export class CommonProgram {
         if (startBlock == 0) {
             startBlock = await this.getBlockHeight()
         }
+        let waitingMinutes = 10
+        let waitingBlocks = 10
```

Comment:
shouldn't waiting blocks be 20? to match with waiting minutes? like below if we can't sign